test/e2e/runs.test.js

```diff
@@ -764,10 +765,11 @@ module.exports = () => {
                     expect(res.body.data.timeO2End).to.equal(dateValue);
                     expect(res.body.data.timeTrgEnd).to.equal(dateValue);
                     expect(res.body.data.runQuality).to.equal('test');
+                    expect(res.body.data.lhcPeriod).to.equal('123123123');
                     done();
                 });
         });
-        it('should be able to update with ', async () => {
+        it('should be able to update with lhc values', async () => {
```

Comment:
The description of the test is not entirely correct as it is updating a run with multiple details

Reply:
Changed the name to `should be able to update with multiple run values`